dashboard/app/config.go

```diff
@@ -108,7 +109,8 @@ type Config struct {
 // subsystem inference / bug reporting.
 type SubsystemsConfig struct {
 	// IMPORTANT: this interface is experimental and will likely change in the future.
-	SubsystemCc func(name string) []string
+	// List stores all subsystems of the namespace.
+	Service *subsystem.Service
```

Comment:
Do we plan to store anything else in this struct in the near future? If not, I would remove SubsystemsConfig and use subsystem.Service directly to not multiply entities unnecessarily.